hw/xbox/xid-sdl.c

```diff
@@ -54,6 +53,9 @@
 #define TYPE_USB_XID_SDL "usb-xbox-gamepad-sdl"
 #define USB_XID(obj) OBJECT_CHECK(USBXIDState, (obj), TYPE_USB_XID_SDL)
 
+#define SDL_MAX_HAPTIC_VALUE 0x7FFF
+#define XID_MAX_HAPTIC_VALUE 0xFFFF
```

Comment:
Confirmed we get a max of `0xFFFF` on both left + right actuator by firing a shotgun in halo